hw/xbox/xid.c

```diff
@@ -158,15 +160,14 @@ static const USBDesc desc_xbox_gamepad = {
 static const XIDDesc desc_xid_xbox_gamepad = {
     .bLength = 0x10,
     .bDescriptorType = USB_DT_XID,
-    .bcdXid = 1,
+    .bcdXid = 0x100,
```

Comment:
Is that what your Duke controller reports? My Controller S and @dracc's Duke report the bcdUSB field as USB 1.1 which for this field, which according to [this](https://www.beyondlogic.org/usbnutshell/usb5.shtml#DeviceDescriptors) would be `0x0110`.